library/src/main/java/org/apache/apex/malhar/lib/window/accumulation/PojoRightOuterJoin.java

```diff
@@ -43,6 +43,12 @@ public PojoRightOuterJoin(int num, Class<?> outClass, String... keys)
 
   }
 
+  public PojoRightOuterJoin(Class<?> outClass, String[] leftKeys, String[] rightKeys)
```

Comment:
Remove previous ctor at line 40... This is the correct one that should nly be  exposed to user,